common/on-chip-calib.cpp

```diff
@@ -406,12 +406,6 @@ namespace rs2
         bool frame_arrived = false;
         try
         {
-            if (_sub->s->supports(RS2_OPTION_THERMAL_COMPENSATION))
```

Comment:
This looks like a mistake
Development has it

Reply:
This merge is scary, need to look on each change against both branches and see what should stay/go

Reply:
?

Reply:
corrected